<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceDialog</class>
 <widget class="QDialog" name="FindReplaceDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>182</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>182</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>481</width>
    <height>182</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Find and Replace</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="FRD_HBoxLayout_find">
     <item>
      <widget class="QLabel" name="FRD_label_findString">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>131</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>Find:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="FRD_lineEdit_findString"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="FRD_Spacer_findreplace">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>360</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="FRD_HBoxLayout_replace">
     <item>
      <widget class="QLabel" name="FRD_label_replaceString">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>131</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>Replace with:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="FRD_lineEdit_replaceString"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="FRD_Spacer_replaceconditions">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>360</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="FRD_GridLayout_conditions">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QCheckBox" name="FRD_checkBox_caseSensitive">
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>240</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>Find case sensitive</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="FRD_label_changeWindow">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>21</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>128</green>
             <blue>64</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>128</green>
             <blue>64</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>127</red>
             <green>127</green>
             <blue>127</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="FRD_checkBox_match">
       <property name="text">
        <string>Must match whole word</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="FRD_Spacer_condtionsbuttons">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>360</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="FRD_HBoxLayout_buttons">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="FRD_Spacer_start">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="FRD_pushButton_replaceAll">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="FRD_Spacer_replaceallpreviews">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="FRD_pushButton_previous">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="FRD_Spacer_previewsnext">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="FRD_pushButton_next">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="FRD_Spacer_nextclose">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>25</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="FRD_pushButton_close">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>FRD_lineEdit_findString</tabstop>
  <tabstop>FRD_lineEdit_replaceString</tabstop>
  <tabstop>FRD_pushButton_next</tabstop>
  <tabstop>FRD_pushButton_close</tabstop>
  <tabstop>FRD_pushButton_previous</tabstop>
  <tabstop>FRD_pushButton_replaceAll</tabstop>
  <tabstop>FRD_checkBox_caseSensitive</tabstop>
  <tabstop>FRD_checkBox_match</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>FRD_pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
